src/Forms/GroupForm/GroupForm.jsx

```diff
@@ -159,6 +164,14 @@ const GroupForm = ({
           }}
           value={name}
         />
+        <TextInput
```

Comment:
maybe we use a multiline input for longer texts

Reply:
Is there an multiline input Component already, or do we still have to do that?

Reply:
This would be perfect :)